oximeter/oximeter/schema/management-network-data-link.toml

```diff
@@ -0,0 +1,140 @@
+format_version = 1
+
+[target]
+name = "management_network_data_link"
+description = "A network data link related to a management gateway virtual local area network (vlan)"
+authz_scope = "fleet"
+
+versions = [
+    { version = 1, fields = [ "kind", "link_name", "model", "network", "rack_id", "sled_id", "sled_model", "sled_revision", "sled_serial", "switch_id", "switch_model", "switch_revision", "switch_serial", "switch_slot", "asic_fab", "asic_lot", "asic_wafer", "asic_wafer_loc_x", "asic_wafer_loc_y" ] },
+]
+
+[fields.kind]
+type = "string"
+description = "The kind or class of the data link (examples: physical-device, gateway, switch-port)"
```

Comment:
Rather than examples, I might put something more generic like, "The kind or class of the data link, such as whether it is a physical or virtual device". It's just a bit less subject to change.